### PR TITLE
chore(vendor): updating scss-bundle

### PR DESCRIPTION
**Pull request by MariaAga**
removing bootstrap mixins, replacing node-sass with sass.

Needed as with npm8 node-sass causes issues, and is anyway deprecated in favor of (dart) sass.
error: `Invalid CSS after "...-padding: (math": expected expression (e.g. 1px, bold), was ...`
Currently `scss-bundle` has an issue where if a few file use `@use` it fails because they are in the middle of the file and not in the beginning: `@use rules must be written before any other rules`.
The `@use` error does not happen if we remove `~bootstrap-sass/assets/stylesheets/bootstrap/mixins`.
we dont use mixins much in foreman and plugins, but in one place we do use a mixin from there I just copied the specific one to foreman-js.

### CODE REVIEW

File: packages/vendor/scripts/build-scss.sh [hunk 1]
```diff
@@ -7,10 +7,10 @@ VENDOR_CORE_PATH=./node_modules/@theforeman/vendor-core
 
 # Build variables
 scss-bundle -e $VENDOR_CORE_PATH/scss/variables.scss \
-            -d scss/variables.scss \
-            --includePaths $VENDOR_CORE_PATH
+            -o scss/variables.scss \
+            -p $VENDOR_CORE_PATH
```
Comment (Ron-Lavi) [inline packages/vendor/scripts/build-scss.sh:11]:
```js
            --project $VENDOR_CORE_PATH // Project location where node_modules is located.
```

File: packages/vendor/scripts/build-scss.sh [hunk 2]
```diff
@@ -7,10 +7,10 @@ VENDOR_CORE_PATH=./node_modules/@theforeman/vendor-core
 
 # Build variables
 scss-bundle -e $VENDOR_CORE_PATH/scss/variables.scss \
-            -d scss/variables.scss \
-            --includePaths $VENDOR_CORE_PATH
+            -o scss/variables.scss \
```
[the remark this answers is not in the document]
Comment (MariaAga) [reply packages/vendor/scripts/build-scss.sh:10]: d was destination :)

**Comment by MariaAga** [comment]
Thanks @Ron-Lavi, I addressed the comments

**Comment by MariaAga** [comment]
Added that this is a breaking change since it removes mixin imports, tests are green @Ron-Lavi

**Review by Ron-Lavi — APPROVED** [review]
Thanks @MariaAga !

**Pull request merged**